tests/installation/hostname_inst.pm

```diff
@@ -27,6 +27,13 @@ sub run {
         # got from environment (DHCP, 'hostname=' as a kernel cmd line argument
         assert_script_run "test \"\$(hostname)\" == \"$expected_install_hostname\"";
     }
+    elsif (check_var("BACKEND", "ipmi")) {
+        my $host_name = script_output "hostname";
+        record_info "$host_name";
+        if ($host_name =~ /(?<host_found>(^openqaipmi5|^fozzie-1))/) {
+            assert_script_run "test \"\$(hostname)\" == \"$+{host_found}\"/";
```

Comment:
Hmm, looks strange, we get hostname from the machine and compare it to the hostname again if it matches one of 2 names?
Having separate clause would help to unify schedules (we have similar issue on hyperV AFAIR), but I would not introduce any checks which either rely on hardcoded values or use data from the machine itself to validate it.
We have `IPMI_HOSTNAME` variable in the job run, can we use that one for validation?